cmd/run.go

```diff
@@ -264,6 +297,8 @@ func dataAccessSync(targetConfig *target.BaseTargetConfig, jobID string, client
 		return err
 	}
 
+	lastAPSync = ptr.Time(time.Now())
```

Comment:
Shouldn't we get this timestamp from the server to be less error-prone?

Reply:
I made this remark as well. I think even better is that we only ask for the APs that are out of sync. I'm working on a follow up PR for this. In that way there is no need to exchange timestamps.

Reply:
Ok, let's plan another issue/PR for this

Reply:
PR for APP server can be found at https://github.com/raito-io/appserver/pull/701